src/resizable-card-layout.scss

```diff
@@ -0,0 +1,282 @@
+@import './new-settings';
+@import './mixins';
+@import './icon';
+
+$block: #{$fd-namespace}-resizable-card-layout;
+$blockContainer: #{$fd-namespace}-card-container;
+
+$fd-grid-column-width: 20rem;
+$fd-grid-row-height: 1rem;
+$fd-grid-gap: 1rem;
+$fd-resize-handle-width: 0.0625rem;
+$fd-border-on-drag: 0.125rem dashed var(--sapContent_DragAndDropActiveColor, #0854a0);
+
+$fd-layout-bottom-padding: 1rem;
+$fd-layout-padding-xl-screen: 3rem;
+$fd-layout-padding: 1rem;
+$fd-layout-padding-s-screen: 0.5rem;
+
+.#{$block} {
+  @include fd-reset();
+
+  @mixin row-span($row-span: 1) {
+    @include fd-reset();
+
+    grid-row-end: span $row-span;
```

Comment:
`grid-row-end` [not supported in IE](https://caniuse.com/?search=grid-row-end). Same with `grid-column-end`